drivers/platform/xilinx/uart.c

```diff
@@ -0,0 +1,400 @@
+/******************************************************************************
+*
+* Copyright (C) 2010 - 2014 Xilinx, Inc.  All rights reserved.
```

Comment:
i would include in the commit message that this patch offers support only for specific xilinx products, not the entire platform.